lib/created_message_queue_handler.ts

```diff
@@ -312,8 +332,10 @@ export function processSuccess(
 /**
  * Handler that gets triggered on incoming event.
  */
-export function index(context: ContextWithBindings): void {
-  // redirect winston logs to Azure Functions log
+export async function index(
+  context: ContextWithBindings
+): Promise<OutputBindings | Error | void> {
+  const stopProcessing = undefined;
```

Comment:
I suggest you return `undefined` and add a comment saying "Returns undefined to stop processing"

Reply:
we can do it but using a variable instead of 'undefined' lets you track where the process exits (while 'undefined' values may show up in some other unrelated places)